src/pages/rest/modules/import/index.md

```diff
@@ -144,3 +155,644 @@ In this example, the CSV payload contains three rows of data, and one of them is
+
+```json
+{
+    "associated_skus": {
+        "Sprite Statis Ball 55mm": 1,
+        "Sprite Foam Yoga Brick": 1,
+        "Sprite Yoga Strap 6 foot": 1
+    }
+}
+```
+
+This structure can be found nested within individual product items in the full payload.
+
+**Payload 6 (Gift card product):**
+
+For a gift card product:
+
+*  The `giftcard_amount` attribute accepts an array of amounts that can be used to purchase the gift card.
+
+```json
+{
+    "giftcard_open_amount_min": 5,
+    "giftcard_open_amount_max": 500,
+    "giftcard_amount": [
+        25,
+        50,
+        75,
+        100
+    ]
+}
+```
+
+This structure can be found nested within individual product items in the full payload.
+
+**Payload 7 (Multiple select attributes):**
+
+Product multiple select attribute is represented as an array of strings.
+
+```json
+{
+    "additional_attributes": {
+        "usage": [
+            "Travel",
+            "Sports",
+            "Office",
+            "Casual"
+        ]
+    }
+}
+```
+
+Customer and customer address multiple select attributes are represented as an array of strings.
+
+```json
+{
+    "interests": [
+        "Electronics",
+        "Sports",
+        "Automotive"
+    ],
+    "delivery_preferences": [
+        "Hand directly to resident",
+        "Leave in the mailbox"
+    ]
+}
+```
+
+Import JSON API does not create attributes automatically. You need to create attributes manually before importing data. For more information, see [Create attributes](https://devdocs.magento.com/guides/v2.4/extension-dev-guide/attributes.html#create-attributes).
+
+**Payload (Advanced Pricing):**
+
+Advanced pricing is represented as an array of JSON objects.
+
+```json
+{
+    "source": {
+        "entity": "advanced_pricing",
+        "behavior": "append",
+        "validation_strategy": "validation-stop-on-errors",
+        "allowed_error_count": 10,
+        "items":	[
+            {
+                "sku": "Simple Product 1",
+                "tier_price_website": "All Websites [USD]",
+                "tier_price_customer_group": "ALL GROUPS",
+                "tier_price_qty": 1,
+                "tier_price": 7.99,
+                "tier_price_value_type": "Fixed"
+            },
+            {
+                "sku": "Simple Product 1",
+                "tier_price_website": "base",
+                "tier_price_customer_group": "NOT LOGGED IN",
+                "tier_price_qty": 2,
+                "tier_price": 40,
+                "tier_price_value_type": "Discount"
+            },
+            {
+                "sku": "Simple Product 1",
+                "tier_price_website": "french",
+                "tier_price_customer_group": "General",
+                "tier_price_qty": 5,
+                "tier_price": 60,
+                "tier_price_value_type": "Discount"
+            }
+        ]
+    }
+}
+```
+
+**Payload (Customer Finances):**
+
+Customer finances is represented as an array of JSON objects.
```

Comment:
```suggestion
`customer_finances` is represented as an array of JSON objects.
```

Reply:
If you propose to use entity name here, it's correct to use `customer_finance` instead of `customer_finances`. Fixed and committed.